qualtran/bloqs/qft/qft_text_book.py

```diff
@@ -56,3 +81,22 @@ def decompose_from_registers(
         if self.with_reverse:
             for i in range(self.bitsize // 2):
                 yield cirq.SWAP(q[i], q[-i - 1])
+
+    def build_call_graph(self, ssa: SympySymbolAllocator) -> Set['BloqCountT']:
+        ret = {
+            (Hadamard(), self.bitsize),
+            (PhaseGradientUnitary(self.bitsize - 1, exponent=0.5, is_controlled=True)),
```

Comment:
this needs to be tuple of `(bloq, n)`. This is missing `n`.

Reply:
whoops yeah I got carried away trying to figure it out from "PhaseGradientUnitary". The current implementation uses PhaseGradientUnitary, which I don't think is a controlled rotation, but I think the number of those bloqs should be `bitsize` - 1?